hasktorch/test/SparseSpec.hs

```diff
@@ -25,13 +25,13 @@ spec = do
     (shape (asTensor v)) `shouldBe` [3]
     print (toDense x)
     -- When we call print for sparse tensor, it throws a exception.
-    (print x) `shouldThrow` anyException
+    print x -- `shouldThrow` anyException
```

Comment:
doesn't throw anymore? did they implement something sparse-related that wasn't implemented before?

Reply:
No. Our understanding is right. They changed the implementation, but I do not find the change.
We can print out sparse-tensor.